test/test_serialize_explanation.py

```diff
@@ -35,21 +35,42 @@ def iris_svm_model(iris):
 @pytest.mark.usefixtures('clean_dir')
 class TestSerializeExplanation(object):
 
-    def test_json_serialize_mimic_no_features(self, iris, iris_svm_model):
-        explainer = MimicExplainer(iris_svm_model,
-                                   iris[DatasetConstants.X_TRAIN],
-                                   LGBMExplainableModel,
-                                   max_num_of_augmentations=10,
-                                   classes=iris[DatasetConstants.CLASSES].tolist())
-        explanation = explainer.explain_global()
-        verify_serialization(explanation)
-
-    def test_json_serialize_local_explanation_classification(self, iris, tabular_explainer, iris_svm_model):
+    def test_save_explanation(self, iris, tabular_explainer, iris_svm_model):
         explainer = tabular_explainer(iris_svm_model,
                                       iris[DatasetConstants.X_TRAIN],
                                       features=iris[DatasetConstants.FEATURES])
         explanation = explainer.explain_local(iris[DatasetConstants.X_TEST])
-        verify_serialization(explanation)
+        save_explanation(explanation, 'brand/new/path')
+
+    def test_save_and_load_explanation_local_only(self, iris, tabular_explainer, iris_svm_model):
+        explainer = tabular_explainer(iris_svm_model,
+                                      iris[DatasetConstants.X_TRAIN],
+                                      features=iris[DatasetConstants.FEATURES])
+        explanation = explainer.explain_local(iris[DatasetConstants.X_TEST])
+        path = 'brand/new/path'
+        save_explanation(explanation, path)
+        loaded_explanation = load_explanation(path)
+        _assert_explanation_equivalence(explanation, loaded_explanation)
+
+    def test_save_and_load_explanation_global_only(self, iris, tabular_explainer, iris_svm_model):
+        explainer = tabular_explainer(iris_svm_model,
+                                      iris[DatasetConstants.X_TRAIN],
+                                      features=iris[DatasetConstants.FEATURES])
+        explanation = explainer.explain_global(iris[DatasetConstants.X_TEST], include_local=False)
+        path = 'brand/new/path'
+        save_explanation(explanation, path)
+        loaded_explanation = load_explanation(path)
+        _assert_explanation_equivalence(explanation, loaded_explanation)
+
+    def test_save_and_load_explanation_global_and_local(self, iris, tabular_explainer, iris_svm_model):
+        explainer = tabular_explainer(iris_svm_model,
+                                      iris[DatasetConstants.X_TRAIN],
+                                      features=iris[DatasetConstants.FEATURES])
+        explanation = explainer.explain_global(iris[DatasetConstants.X_TEST], include_local=False)
+        path = 'brand/new/path'
```

Comment:
there seems to be a bit of repetition here in the tests, consider refactoring to a helper util